src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -4209,6 +4212,24 @@ def test_sql_managed_instance_mgmt(self, mi, rg):
                      JMESPathCheck('resourceGroup', resource_group_1),
                      JMESPathCheck('tags', {})])
 
+        # test cross-subnet update SLO. Since the feature isn't rolled out, we expect the operation to fail.
```

Comment:
When will the feature get released? Will anybody update the tests after feature rolled out?